### ISSUE TITLE
Feature Request: Add SAS/SATA/NVMe "SANITIZE" command option for secure erasure

### ISSUE THREAD

**Issue by MrPippin66** [storaged-project/udisks#1235]
I see no option in the current code for anything beyond the "SECURE ERASE" command invocation.

Per the NIST standard (NIST Special Publication 800-88, Revision 1, Guidelines for Media Sanitization), it's preferred to use the 'SANITIZE" command over "SECURE ERASE" where available.

```Use the ATA Security feature set’s SECURE ERASE UNIT command, if support, in
Enhanced Erase mode. The ATA Sanitize Device feature set commands are preferred
over the over the ATA Security feature set SECURITY ERASE UNIT command when
supported by the ATA device.

I would request this option be supported in a future release. 

It would also be advisable to allow Cryptographic Erase, where SED has been used.

**Comment by tbzatek** [comment]
Agree, this would be useful. No promises when this actually happens, patches welcome.

The NVMe Sanitize command is available since the last release. Let us know if it fits your needs. Support for additional TPs like TP4152 may be added in the future.

There's currently no specific support for SAS drives (like the `org.freedesktop.UDisks2.Drive.Ata` interface). #1007 will be needed first.

Also, there's been a similar secure erase request, although that's about overwrite methods: #971

**Comment by MrPippin66** [comment]
Thanks!

Overall is supporting the same "sanitize" command function on the major protocols (SAS/SATA/NVMe).

The chain dependency is understandable.
